### PR TITLE
Post react release 20221108

### PR DESCRIPTION
**Pull request by dmichaels-harvard**

### CODE REVIEW

File: chalicelib_fourfront/checks/helpers/wfr_utils.py [hunk 1]
```diff
@@ -178,6 +178,8 @@
     'TrAC-loop':     ["HiC_Pipeline_0.2.6", "HiC_Pipeline_0.2.7"],
     # Preliminary - Released to network  # NO-NORM
     'PLAC-seq':      ["HiC_Pipeline_0.2.6", "HiC_Pipeline_0.2.7"],
+    # Preliminary - Released to network  # NO-NORM
```
Comment (dmichaels-harvard) [inline chalicelib_fourfront/checks/helpers/wfr_utils.py:181]: I do not understand why this diff is showing. This file is the same as what's in the master branch.

File: chalicelib_fourfront/checks/wfr_checks.py [hunk 2]
```diff
@@ -1254,6 +1254,61 @@ def plac_seq_start(connection, **kwargs):
     action = wfr_utils.start_tasks(missing_runs, patch_meta, action, my_auth, my_env, fs_env, start, move_to_pc=True)
     return action
 
+@check_function(lab_title=None, start_date=None)
```
Comment (dmichaels-harvard) [inline chalicelib_fourfront/checks/wfr_checks.py:1257]: I do not understand why this diff is showing. This file is the same as what's in the master branch.

File: chalicelib_fourfront/check_setup.json [hunk 3]
```diff
@@ -1291,6 +1291,21 @@
             }
         }
     },
+    "hichip_status": {
```
Comment (dmichaels-harvard) [inline chalicelib_fourfront/check_setup.json:1294]: I do not understand why this diff is showing. This file is the same as what's in the master branch.

File: chalicelib_fourfront/app_utils.py [hunk 4]
```diff
@@ -26,11 +26,6 @@ class AppUtils(AppUtils_from_core):
     host = HOST
     package_name = 'chalicelib_fourfront'
 
-    check_setup_file = AppUtils_from_core.locate_check_setup_file(os.path.dirname(__file__))
```
Comment (dmichaels-harvard) [inline chalicelib_fourfront/app_utils.py:29]: Lookup now done in foursight-core.

File: chalicelib_fourfront/accounts.json [hunk 5]
```diff
@@ -0,0 +1 @@
+9NovjmdNf+ITE/xELDWvv8CYlsXKvWDEy3MXV4fk5VvRX0fBWTRuvrRX5W8KzwvZGCzdqfzIPjrrEPWWMaDjEbuFl9aZ1uZJSO7NigVjl6yczgww2Vj0/HlnG75vsoeES5lLMv0BoQYi8lpWxHJJWOQHbYUq1EAkAknzJKRUemFbYPVwu2NADOXLOReCVvlzgJpvfERwpFU4wFghf/q3LYHD03PivsrcH+A9yFvc+b2MGzhZGAwfjrk7X/l8f+kf4xVFSaXvaU2s5yEHgfsWwKQt/9mPPuCMu4WX1pnW5klI7s2KBWOXrJzODDDZWPT8Nbkki4cO7mfZDqF6CB8p4CLyWlbEcklY5AdthSrUQCQCSfMkpFR6YVtg9XC7Y0AM5cs5F4JW+XNJCDhSsCEzUvrkWeFmBdKR
```
Comment (netsettler) [inline chalicelib_fourfront/accounts.json:1]: I would favor this file being called `accounts.json.encrypted`.  Or something like that. I don't see how the contents of this file are JSON.

**Review by netsettler — APPROVED** [review]
No actual blockers, though hoping the suggestion of renaming `accounts.json` can be taken.

**Pull request merged**